docs/content/preview/develop/pg15-features.md

```diff
@@ -0,0 +1,184 @@
+---
+title: PostgreSQL 15 features
+headerTitle: PostgreSQL 15 features
+linkTitle: PG15 features
+description: Use PostgreSQL 15 features in your applications
+headContent: Use PostgreSQL 15 features in your applications
+menu:
+  preview:
+    identifier: yb-postgresql-15
+    parent: develop
+    weight: 700
+type: docs
+---
+
+The YugabyteDB [YSQL API](../../api/ysql/) reuses a fork of the PostgreSQL [query layer](../../architecture/query-layer/). This architecture allows YSQL to support most PostgreSQL features, such as data types, queries, expressions, operators and functions, stored procedures, triggers, extensions, and so on - all of which are expected to work identically on both database systems.
+
+Initially based on PostgreSQL 11.2, YSQL has been continuously enhanced with features, improvements, and security fixes pulled from PostgreSQL 12+, in addition to our own improvements. With {{<release "2.25">}}, YugabyteDB now adds support for PostgreSQL 15, which brings many new features and improvements.
+
+{{<lead link="https://www.yugabyte.com/blog/yugabytedb-moves-beyond-postgresql-11/">}}
+Learn more about the [journey to PostgreSQL 15](https://www.yugabyte.com/blog/yugabytedb-moves-beyond-postgresql-11/).
+{{</lead>}}
+
+## What's new
+
+### Major features
+
+{{%table%}}
+
+| Feature | Description |
+| --------| ----------- |
+
+| [Stored generated columns](../../api/ysql/the-sql-language/statements/ddl_create_table#stored-generated-columns)
+| Define columns whose values are automatically calculated and stored based on expressions involving other columns.
+|
+
+|[Before&nbsp;row&nbsp;triggers&nbsp;on partitioned tables](../../api/ysql/the-sql-language/statements/ddl_create_trigger#partitioned-tables)
+| Enforce custom business logic, perform validation, or even modify row data before it is written to the appropriate partition.
+|
+
+| [Non distinct NULLs in a Unique index](../../api/ysql/the-sql-language/statements/ddl_create_index#nulls-not-distinct)
+| Enforce that only one NULL value is permitted instead of allowing multiple NULL entries in a unique index.
+|
+
+| [Filter rows while importing data](../../api/ysql/the-sql-language/statements/cmd_copy#where)
+| Use COPY FROM with WHERE clause while importing data into a table and eliminate preprocessing data files.
+|
+
+| [UUID generation](../../api/ysql/exprs/func_gen_random_uuid/)
+| Provide `gen_random_uuid` natively, eliminating the need for external libraries or custom implementations to generate UUIDs.
+|
+
+| [Foreign keys on partitioned tables](../../explore/ysql-language-features/advanced-features/partitions/#foreign-key-references)
+| Enforce referential integrity directly on partitioned tables, ensuring consistency across large-scale datasets that benefit from partitioning for performance and scalability.
```

Comment:
Specifying foreign key constraints on a partitioned table was supported in PG11.2 and YB-before-pg15-merge. However, foreign key constraints could not reference a partitioned table. This is now supported. So it might be more clear if we reword "Foreign keys on partitioned tables" to "Foreign keys referencing partitioned tables". 

I think the explanation "Enforce referential integrity directly on partitioned tables ..." will also need some change along the same lines.

Reply:
Alternately, the title could be "Foreign Key references for partitioned tables" with the explanation "A foreign key can reference a partitioned table." I copied it from PG https://www.postgresql.org/about/featurematrix/detail/319/

Reply:
Fixed.